src/Sparrow.Server/LowMemory/MemoryInformation.cs

```diff
@@ -225,7 +225,8 @@ public static MemoryInfoResult GetMemoryInformationUsingOneTimeSmapsReader()
             }
         }
 
-        internal static (Size MemAvailable, Size TotalMemory, Size Commited, Size CommitLimit, Size AvailableWithoutTotalCleanMemory, Size SharedCleanMemory) GetFromProcMemInfo(SmapsReader smapsReader)
+        internal static (Size MemAvailable, Size TotalMemory, Size Commited, Size CommitLimit, Size AvailableWithoutTotalCleanMemory, Size SharedCleanMemory, Size TotalDirty)
```

Comment:
Let's make a proper struct of this at this point and accept it as a `ref` parameter

Reply:
https://github.com/ravendb/ravendb/pull/9721